components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -2535,7 +2535,7 @@ private APITemplateBuilder getAPITemplateBuilder(API api) throws APIManagementEx
                     , Collections.<String, String>emptyMap());
 
             properties = new HashMap<String, String>();
-            properties.put("configKey", "gov:" + APIConstants.GA_CONFIGURATION_LOCATION);
+            properties.put("configKey", APIConstants.GA_CONF_KEY);
```

Comment:
Can we use a constang for "configKey" as well, since its used in many places

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/9250